wsd-frontend/src/lib/hooks.ts

```diff
@@ -145,3 +145,134 @@ export function useEffectAfterMount(effect: EffectCallback, deps: DependencyList
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, deps)
 }
+
+type UseFilePasteOptions = {
```

Comment:
Can you inline these as these are one-off types and wont be used in a context where they won't be able to inferred